src/Plivo/Resources/Message/MessageInterface.php

```diff
@@ -144,66 +132,101 @@ protected function getList($optionalArgs = [])
      * @throws PlivoValidationException,PlivoResponseException
      */
 
-    public function create($src, array $dst, $text=null,
-                           array $optionalArgs = [], $powerpackUUID = null)
+    public function __call($name_of_method, $arguments)
```

Comment:
why there is function signature change?  @nixonsam  Please check once

Reply:
is there respective unit case changes also done?

Reply:
__call() is used here for method overloading. No changes required to update the test case.